docs/deployment/index.md

```diff
@@ -0,0 +1,46 @@
+# Deploying Connect
+
+_Written by @szymonlesisz, checked by @tsusanka._
+
+
+## Creating new minor version
+
+1. Go to develop branch.
+1. Make sure that CHANGELOG is up to date, usually it is not so walk thru commits and update it.
+1. Run `make version-patch` - this will update version number in multiple files.
+1. Commit these changes with the Changelog changes into `develop` with message "version X.X.X" ([example commit](https://github.com/trezor/connect/commit/d28e506524bc666757715294f7d030ea1a1d6eef)).
+1. Prepare PR from `develop` to `v8` branch, paste changelog as description and tag @prusnak or @tsusanka ([example PR](https://github.com/trezor/connect/pull/812)).
+1. Review and merge.
+
+
+## Deploying
+1. After the merge we deploy Connect to connect.trezor.io. TODO description by @prusnak.
```

Comment:
@prusnak could you please describe what steps need to be done to deploy Connect? Just dumping it in comment is fine, I'll add it.

Reply:
to deploy version 8:

```
git checkout v8
make clean
make build
make sync-8
```

Reply:
Thanks. Added.